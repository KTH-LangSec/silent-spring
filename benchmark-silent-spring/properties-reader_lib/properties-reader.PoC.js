//https://snyk.io/vuln/SNYK-JS-PROPERTIESREADER-1048968
  const propertiesReader = require("properties-reader");
  const path = require("path");

  propertiesReader(path.resolve(__dirname, "./payload.ini"));
