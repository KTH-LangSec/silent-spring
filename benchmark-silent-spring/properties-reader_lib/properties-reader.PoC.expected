node_modules/properties-reader/src/properties-reader.js:200:27